### PR TITLE
Improve header layout

### PR DESCRIPTION
**Pull request by maysakanoni**
**What**  
Adds bottom margin for resources which contain a summary bellow the heading. This was removed by the previous pr as summary information outside the food resources were not accounted for. 
image

**Why**  
For consistency and to improve readability and user experience.

### CODE REVIEW

File: components/Feature/VulnerabilitiesGrid/ResourceCard/index.js [hunk 1]
```diff
@@ -33,7 +32,8 @@ const ResourceCard = ({
   const distributionElement =  tags.filter(t => HIDDEN_TAGS.includes(t)).join(", ")
   const tagsElement = tags.filter(t => !HIDDEN_TAGS.includes(t)).map(item=> (<span key={"tags-"+item} className={css.tags}>{trimLength(item, 20)}</span>))
   const snapshot = (customerId != undefined) ? true : false
-  const foodResource = (foodNeedsId == 'food-needs') ? true : false
+  const summaryDataExists = ( telephone != undefined || distance != 100 || currentProvision != undefined || openingTimes != undefined) ? true : false
```
Comment (kjdchapman) [inline components/Feature/VulnerabilitiesGrid/ResourceCard/index.js:35]: My js-fu is weak, but from playing in a [REPL](https://repl.it/languages/javascript) I think testing for `undefined` doesn't work with this syntax?

Are we actually checking for [undefined, or null](https://stackoverflow.com/questions/5076944/what-is-the-difference-between-null-and-undefined-in-javascript)?

Comment (maysakanoni) [reply components/Feature/VulnerabilitiesGrid/ResourceCard/index.js:35]: good shout, i think it's null. I wonder why it still works

File: components/Feature/VulnerabilitiesGrid/ResourceCard/index.js [hunk 2]
```diff
@@ -33,7 +32,8 @@ const ResourceCard = ({
   const distributionElement =  tags.filter(t => HIDDEN_TAGS.includes(t)).join(", ")
   const tagsElement = tags.filter(t => !HIDDEN_TAGS.includes(t)).map(item=> (<span key={"tags-"+item} className={css.tags}>{trimLength(item, 20)}</span>))
   const snapshot = (customerId != undefined) ? true : false
-  const foodResource = (foodNeedsId == 'food-needs') ? true : false
+  const summaryDataExists = ( telephone  || distance != 100 || currentProvision || openingTimes) ? true : false
```
Comment (kjdchapman) [inline components/Feature/VulnerabilitiesGrid/ResourceCard/index.js:35]: I think you can do this without the `? true : false` on the end, as the first part of it already gives you a true/false?

```
> 1 + 1 = 2
true
> 1 + 1 = 2 ? true : false
true
```

File: components/Feature/VulnerabilitiesGrid/index.js [hunk 3]
```diff
@@ -324,7 +324,6 @@ const VulnerabilitiesGrid = ({ resources, onUpdate, residentCoordinates, generic
                               {...resource}
                               updateSelectedResources = {updateSummaryResource}
                               customerId={customerId}
-                              foodNeedsId = {id}
```
Comment (kjdchapman) [inline components/Feature/VulnerabilitiesGrid/index.js:327]: I don't understand this removal as part of this PR - I think I'm missing something?

**Review by kjdchapman — APPROVED** [review]
Have a few small questions still, but nothing that should block this one, I think.

File: components/Feature/VulnerabilitiesGrid/index.js (hunk 3 above)
Comment (maysakanoni) [reply components/Feature/VulnerabilitiesGrid/index.js:327]: Originally (in the previous pr), I was doing the styling based on whether or not it was a food category. The assumption there was that summaries only existed for this category. That not correct, so now I'm checking if any of those summaries (telephone, distance etc )are included which is now what determined when the styling should be applied.

Comment (kjdchapman) [reply components/Feature/VulnerabilitiesGrid/index.js:327]: Ah, OK, so this now isn't needed for this, I understand.

**Review by kjdchapman — APPROVED** [review]
(I know I already approved, but I am happy for this one to be rebased in whenever!)

**Pull request merged**